### PR TITLE
Added “whats-new-in-swift-4” library

### PR DESCRIPTION
**Pull request by Yogesh-MV**
## Project URL
https://github.com/ole/whats-new-in-swift-4

## Description

 An Xcode playground showcasing the new features in Swift 4.0. https://oleb.net

## Checklist

- [X] Only one project/change is in this pull request
- [X] Addition in chronological order (bottom of category)
- [ ] Supports iOS 9 / tvOS 10 or later
- [X] Supports Swift 3 or later
- [X] Has a commit from less than 2 years ago
- [X] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 3 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
175 | [302](https://httpstatuses.com/302) | http://lifehacker.com/i-want-to-write-ios-apps-where-do-i-start-1644802175 redirects to<br>https://lifehacker.com/i-want-to-write-ios-apps-where-do-i-start-1644802175
1647 | [301](https://httpstatuses.com/301) | https://github.com/poolqf/FillableLoaders redirects to<br>https://github.com/polqf/FillableLoaders
2639 | [302](https://httpstatuses.com/302) | http://lifehacker.com/tag/ios redirects to<br>https://lifehacker.com/tag/ios


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by lfarah** [comment]
What do you guys think? @dkhamsing @vsouza @victorgama ?

**Comment by dkhamsing** [comment]
Looks good

File: README.md [hunk 1]
```diff
@@ -2616,6 +2616,8 @@ Most of these are paid services, some have free tiers.
 * [Objective-C Cheat Sheet](https://github.com/iwasrobbed/Objective-C-CheatSheet) - A quick reference cheat sheet for common, high level topics in Objective-C.
 * [SwiftSnippets](https://github.com/hyperoslo/SwiftSnippets) - A collection of Swift snippets to be used in Xcode
 * [App Store Checklist](https://github.com/whitef0x0/app-store-checklist) - A checklist of what to look for before submitting your app to the App Store.
+* [whats-new-in-swift-4](https://github.com/ole/whats-new-in-swift-4) - An Xcode playground showcasing the new features in Swift 4.0. https://oleb.net.
+
```
Comment (lfarah) [inline README.md]: Please remove this extra line

**Pull request merged**